pennylane_lightning/lightning_base/lightning_base.py

```diff
@@ -173,18 +181,22 @@ def execute(
             TensorLike, tuple[TensorLike], tuple[tuple[TensorLike]]: A numeric result of the computation.
         """
 
-    @abstractmethod
     def simulate(
         self,
         circuit: QuantumScript,
         state,  # Lightning [Device] StateVector
+        *,
+        mcmc: dict = None,
         postselect_mode: str = None,
```

Comment:
```suggestion
        postselect_mode: str = None,
        mcmc: dict = None,
```

I suggest to swap them because `postselect_mode` is present in all devices. Also, we should change the name to something more descriptive e.g. `mcm_postselect_mode` or `mid-circ-measure-postselect_mode` but could be done in another PR :smile:

Reply:
Let's get back to the name of this attribute in your TT-MCM PR :)